.changeset/clever-badgers-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@effection/node": major
```

Comment:
```suggestion
"@effection/node": minor
```
Even though this is a breaking change, I think we want to stay pre 1.0 for now since we may want to re-visit the whole `ChildProcess` api or bring it into its own package